### PR TITLE
Made grid component

### PR DESCRIPTION
**Pull request by poojavishnoi**

### CODE REVIEW

File: components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.grid_container {
+  margin: 1rem 0.5rem;
+}
+
+.grid_main_container {
+  padding: 2rem;
+  margin: 1rem;
+  border: 1px solid lightgrey;
```
Comment (AtulHande03a) [inline components/Grid/grid.css]: use  variables instead of color names

File: components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  padding: 0 2rem;
+}
+
+.grid_content button {
+  padding: 0.5rem;
+  font-size: 1.1rem;
+  border: 1px solid lightgrey;
+  cursor: pointer;
+  color: var(--primary-color);
+  font-family: "Roboto", sans-serif;
```
Comment (AtulHande03a) [inline components/Grid/grid.css]: why is the font-family specifically mentioned, does the body have different font-family??

**Review by AtulHande03a — APPROVED** [review]
apart from variable names for colors and font-family name for a specific grid , rest all looks good

File: components/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  align-items: center;
+  gap: 1rem;
+  text-align: center;
+  padding: 0 2rem;
+}
+
+.grid_content button {
+  padding: 0.5rem;
+  font-size: 1.1rem;
+  border: 1px solid lightgrey;
```
Comment (anandkumarcoder) [inline components/Grid/grid.css]: same as above, lightgrey ? if using color name across your code base then pls ignore it

File: doc/doc.html [hunk 4]
```diff
@@ -762,6 +762,114 @@ <h3>Shop with exclusive offers</h3>
...
+            class="img_responsive"
+            src="../images/profile1.jpg"
+            alt=""
+          />
+        </div>
+        <div class="grid_image">
+          <img
+            class="img_responsive"
+            src="../images/profile2.jpg"
+            alt=""
```
Comment (anandkumarcoder) [inline doc/doc.html:850]: try to add values in alt do same for above as well

File: components/Grid/grid.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+    <p>
+      Add grid_4_column layout class on wrapper div which is wrapping 2 divs
+    </p>
+    <div class="grid_main_container grid_4_column">
+      <div class="grid_image">
+        <img
+          class="img_responsive"
+          src="https://cdn.pixabay.com/photo/2021/11/25/19/50/tape-6824489__340.jpg"
+          alt=""
```
Comment (anandkumarcoder) [inline components/Grid/grid.html:73]: add values to alt

**Review by anandkumarcoder — APPROVED** [review]
few small changes, rest all looks good.

**Pull request merged**